Dockerfile

```diff
@@ -1,60 +1,35 @@
 ARG ALPINE_VERSION="3.17"
```

Comment:
It is important for the project to keep track of the specific Tor version we're using. Since we are installing tor with the package manager, I'd like ALPINE_VERSION to specify the minor version `3.17.3`